### PR TITLE
Enable calling 'send' function with jest

### PR DESCRIPTION
**Pull request by kenjiokabe**
fixed 'send' function bug.
( under the condition " jest + superagent-mock ", send function makes an Error.
  Tests passes, but timeout error occurrs after seconds )

### CODE REVIEW

**Comment by oziks** [comment]
Your proposal isn't in the open source spirit. You have changed the author name and the package name.

We can not merge your work!

**Comment by kenjiokabe** [comment]
@oziks I'm very sorry that I didn't understand open source ...
I reverted author name and the package name.
Please reconsider to merge my commit.

File: tests/support/expectations.js [hunk 1]
```diff
@@ -96,6 +96,17 @@ module.exports = function (request, config) {
             test.equal(result, 'Real call done');
             test.done();
           });
+      },
+
+      'also can use "send" method': function (test) {
+        request.get('https://domain.example/666')
+          .send({superhero: "me"})
```
Comment (fdubost) [inline tests/support/expectations.js]: You don't check that this parameter is passed to the fixture function :snowman:

File: lib/superagent-mock.js [hunk 2]
```diff
@@ -42,6 +43,21 @@ function mock (superagent, config) {
...
 
   /**
+   * Override send function
+   */
+  Request.prototype.send = function (data) {
+
+    var parser = parsers[this.url];
+    if (parser) {
+      this.params = data;
+      return this;
```
Comment (fdubost) [inline lib/superagent-mock.js]: Add an empty line before `return` statement please :wink:

**Comment by fdubost** [comment]
Can you revert the permission changes of `LICENCE`and `README` files ? Can you also add some doc about `send` mock in the README ? :relaxed:

**Comment by kenjiokabe** [comment]
@fdubost Thank you for your comments.
I fixed those.
Could you check my commits?

**Comment by fdubost** [comment]
Thanks for these corrections. Could you still :
* revert permission changes of `package.json`
* add some doc about `send` mock in the README
* squash your commits

Thank you very much :smiley:

**Pull request closed without merging**